frontend/src/components/shared/HelpButton.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <button type="button" class="helpCircleButton" @click="redirectToPage()">
```

Comment:
I checked the issue using the WAVE accessibility evaluation tool (https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) and it doesn't like the empty button, meaning the button has no text. Using:
aria-label='info button' 
inside button tag will fix that issue